dropwizard-jetty/src/test/java/io/dropwizard/jetty/HttpsConnectorFactoryTest.java

```diff
@@ -20,10 +20,12 @@
 import org.eclipse.jetty.util.thread.ScheduledExecutorScheduler;
 import org.eclipse.jetty.util.thread.ThreadPool;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.OS;
```

Comment:
This can go now actually, it got imported statically in another PR. I'll rebase it out.